src/views/List.vue

```diff
@@ -15,28 +43,52 @@ import ListItem from '@/components/ListItem.vue';
 export default {
   name: 'List',
   props: {
-  title: {
-    type: String,  // vue check to see if the type you passed is the expected type
-    default: "New List",  //the default value. if the type is Object, this MUST use a function
-    description: 'The title of the list you are displaying. Defaults to "New List".'
-    }
+    title: {
+      type: String, // vue check to see if the type you passed is the expected type
+      default: 'New List', // the default value. if the type is Object, this MUST use a function
+      description: 'The title of the list you are displaying. Defaults to "New List".',
+    },
   },
   components: {
     ListItem,
   },
+  data() {
+    return {
+      entry: '',
+    };
+  },
   computed: {
     ...mapGetters(['list']),
     theList() {
```

Comment:
Please refrain from using this pattern. Do not have two variables that are the same. Is one a list, and the other a specific list? If so, how is it different?

Reply:
theList returns a List which is going to be use inside listItems.
Inside theList, for more clarity wantedList was defined in order not to repeat wantedList.listItems.length
Could you please elaborate on what you are asking me to do?